docs/overview/header.md

```diff
@@ -5,3 +5,74 @@ to the top of the viewport even when the main content area is scrolled up and do
 In order to create a header you have to create a CDI bean that implement org.uberfire.client.workbench.Header interface.
 
 ![header](header.png)
+
+Let us add a simple Composite that adds 3 labels. 
+
+In package org.uberfire.client.navbar we can create 
+
+- HeaderView.java
+
+```
+package org.uberfire.client.navbar;
+
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.HorizontalPanel;
+import com.google.gwt.user.client.ui.Label;
+
+public class HeaderView extends Composite {
+
+    public HeaderView() {
```

Comment:
I am trying to use an MVP pattern
But I do not know how to fit all together.
In the example you generate, the presenter are created by uferfire and only used when in the perspective (so they hall be ready)
In which order all this is created ?
I try to do this 
public class AppNavBar
        extends Composite implements Header {

    @Inject
    private WorkbenchMenuBarPresenter menuBarPresenter;

    @Inject
    HeaderViewPresenter headerViewPresenter;

    @Override
    public Widget asWidget() {
        this.headerViewPresenter.show(menuBarPresenter);
        return headerViewPresenter.getView().asWidget();
    }
but in the browser  I get a 
"Uncaught java.lang.AssertionError: This UIObject's element is not set; you may be missing a call to either Composite.initWidget() or UIObject.setElement()"
so it means cdi miracle did not create all stuff :)
How can I do ?